packages/extension/src/background/index.ts

```diff
@@ -50,23 +50,7 @@ browser.alarms.onAlarm.addListener(async (alarm) => {
 
   if (alarm.name === transactionTrackerUpdateAlarm) {
     console.info("~> fetching transaction updates")
-    let hasInFlightTransactions = await transactionTracker.update()
-
-    // the config below will run transaction updates 7x per minute, if there are in-flight transactions
-    // it will update on second 0, 15, 30 and 45
-    const maxRetries = 6 // max 3 retries
-    const waitTimeInS = 7.5 // wait 7.5 seconds between retries
-
-    let runs = 0
-    while (hasInFlightTransactions && runs < maxRetries) {
-      console.info(`~> waiting ${waitTimeInS}s for transaction updates`)
-      await delay(waitTimeInS * 1000)
-      console.info(
-        "~> fetching transaction updates as pending transactions were detected",
-      )
-      runs++
-      hasInFlightTransactions = await transactionTracker.update()
-    }
+    await transactionTracker.update()
```

Comment:
Btw, does this update both mainnet and testnet?

Reply:
Yes, it updates each transaction status based on the network id.

Reply:
Perfect.